app/mailers/user_mailer.rb

```diff
@@ -51,8 +51,7 @@ def notification_received(user, ssr)
   private
 
   def send_message subject, is_service_provider='false', ssr_id=''
-    email = Rails.env == 'production' ? @send_to.email : DEFAULT_MAIL_TO
-    subject = Rails.env == 'production' ? subject : "[#{Rails.env.capitalize} - EMAIL TO #{@send_to.email}] #{subject}"
+    email = @send_to.email
```

Comment:
Yes intentional, subject is passed straight through to mail.

Otherwise it would have been `subject = subject` I think